apps/worker/src/app/workflow/usecases/send-message/send-message.usecase.ts

```diff
@@ -206,34 +207,19 @@ export class SendMessage {
     command: SendMessageCommand,
     variables: IFilterVariables
   ): Promise<{ filterResult: IConditionsFilterResponse | null; channelPreferenceResult: boolean | null }> {
-    const skipHalt = this.shouldSkipHalt(bridgeSkip, command.job?.step?.bridgeUrl);
-    if (skipHalt) {
+    if (bridgeSkip === true) {
       return { filterResult: { passed: true, conditions: [], variables: {} }, channelPreferenceResult: true };
     }
 
     const [filterResult, channelPreferenceResult] = await Promise.all([
-      this.filter(command, variables),
-      this.filterPreferredChannels(command.job),
+      this.filterConditions(command, variables),
```

Comment:
Renamed `this.filter` -> `this.filterConditions` to create a distinction between the two types of filters evaluated. One is Step conditions, the other is Delivery conditions.

Reply:
Is the first filter about skip conditions? If yes I'd say we can do `this.filterSkipConditions` to be more precise.

Reply:
Yes, it is. I renamed again to be more accurate, we now have:
```typescript
    const [stepCondition, channelPreference] = await Promise.all([
      this.evaluateStepCondition(command, variables),
      this.evaluateChannelPreference(command),
    ]);
```